plugins/rikki/heroeslounge/components/UpcomingMatches.php

```diff
@@ -84,25 +85,39 @@ public function collectMatches($timezoneName, $id)
         }
         if ($myEntity) {
             if ($type == 'all') {
-                $matches = Matches::with('teams', 'teams.logo', 'casters', 'channels', 'division')->where('winner_id', null)->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->orderBy('wbp', 'asc')->get();
+                $matches = Matches::with('teams', 'teams.logo', 'casters', 'channels', 'division')->where('winner_id', null)->orderBy('wbp', 'asc')->orderBy('id', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
             } elseif ($type == 'caster') {
                 if ($this->idApp == 'denied') {
-                    $matches = $myEntity->castMatches()->where('rikki_heroeslounge_match_caster.approved', '=', '2')->where('winner_id', null)->orderBy('wbp', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
+                    $matches = $myEntity->castMatches()->where('rikki_heroeslounge_match_caster.approved', '=', '2')->where('winner_id', null)->orderBy('wbp', 'asc')->orderBy('id', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
                 } elseif ($this->idApp == 'accepted') {
-                    $matches = $myEntity->castMatches()->where('rikki_heroeslounge_match_caster.approved', '=', '1')->where('winner_id', null)->orderBy('wbp', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
+                    $matches = $myEntity->castMatches()->where('rikki_heroeslounge_match_caster.approved', '=', '1')->where('winner_id', null)->orderBy('wbp', 'asc')->orderBy('id', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
                 } elseif ($this->idApp == 'pending') {
-                    $matches = $myEntity->castMatches()->where('rikki_heroeslounge_match_caster.approved', '=', '0')->where('winner_id', null)->orderBy('wbp', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
+                    $matches = $myEntity->castMatches()->where('rikki_heroeslounge_match_caster.approved', '=', '0')->where('winner_id', null)->orderBy('wbp', 'asc')->orderBy('id', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
                 } else {
-                    $matches = $myEntity->castMatches()->orderBy('wbp', 'asc')->where('winner_id', null)->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
+                    $matches = $myEntity->castMatches()->where('winner_id', null)->orderBy('wbp', 'asc')->orderBy('id', 'asc')->where('wbp', '>=', Carbon::today())->where('wbp', '<=', Carbon::today()->addDays($daysInFuture))->get();
```

Comment:
This is mostly reordering the query builder statements to be consistent, but I also added a secondary `orderBy` on id, since otherwise the two playoff rounds with the same start time could be out of order.E.g: Round 2 would be listed before Round 1 in the calendar.